utils/idMapping.js

```diff
@@ -0,0 +1,37 @@
+const { fetchUser } = require('../models/users')
+
+/**
+ * Used for receiving userId when providing username
+ *
+ * @param username {String} - username of the User.
+ * @returns id {String} - userId of the same user
+ */
+const toUserId = async (username) => {
```

Comment:
Can we simply name the functions as getUserId and getUsername?

toUserId seems fine when its a class method, but here, its a function.